pysat/_files.py

```diff
@@ -912,6 +912,18 @@ def from_os(cls, data_path=None, format_str=None,
         part of the name that need not be extracted.
         'cnofs_cindi_ivm_500ms_{year:4d}{month:02d}{day:02d}_v??.cdf'
 
+        Leading wilcards are supported when parsing fixed width filenames,
+        '*{year:4d}{month:02d}{day:02d}_v??.cdf', though the '*' is not
```

Comment:
Make this an example that would work.

Reply:
That works.

Reply:
Not according to the words following this.  If this does work, the next sentence needs to be changed.

Reply:
I reorganized the first sentence of the paragraph and such to hopefully make the description of the two different parsers clearer. I also added a comment in the delimited section about '?' which now works for that parser.